pkgs/tools/text/ripgrep/default.nix

```diff
@@ -37,6 +37,14 @@ rustPlatform.buildRustPackage rec {
     installShellCompletion --zsh complete/_rg
   '';
 
+  doInstallCheck = true;
+  installCheckPhase = ''
```

Comment:
Maybe for a future PR, but I guess it'd also be nice to check `rg` with `-P`, since PCRE2 is linked externally (to detect any regressions related to that dependency).